superset-frontend/src/features/databases/CSVUploadModal/index.tsx

```diff
@@ -0,0 +1,805 @@
+                >
+                  <Switch data-test="dataFrameIndex" />
+                </StyledFormItem>
+                <p className="help-block">
+                  {t('Write dataframe index as a column')}
+                </p>
+              </Col>
+            </Row>
+            <Row>
+              <Col span={24}>
+                <StyledFormItem
+                  label={t('Column Label(s)')}
+                  name="column_labels"
+                >
+                  <Input aria-label={t('Column labels')} type="text" />
+                </StyledFormItem>
+                <p className="help-block">
+                  {t(
+                    'Column label for index column(s). If None is given and Dataframe Index is checked, Index Names are used',
+                  )}
+                </p>
+              </Col>
+            </Row>
+            <Row>
+              <Col span={24}>
+                <StyledFormItem
+                  label={t('Columns To Read')}
+                  name="columns_read"
+                >
+                  <Select
+                    ariaLabel={t('Choose columns to read')}
+                    mode="multiple"
+                    options={columnsToOptions()}
+                    allowClear
+                    allowNewOptions
+                  />
+                </StyledFormItem>
+                <p className="help-block">
+                  {t('List of the column names that should be read')}
+                </p>
+              </Col>
+            </Row>
+            <Row>
+              <Col span={24}>
+                <StyledFormItem
+                  label={t('Overwrite Duplicate Columns')}
+                  name="overwrite_duplicates"
+                >
+                  <Switch data-test="overwriteDuplicates" />
+                </StyledFormItem>
+                <p className="help-block">
+                  {t(
+                    'If duplicate columns are not overridden, they will be presented as "X.1, X.2 ...X.x"',
+                  )}
+                </p>
+              </Col>
+            </Row>
+            <Row>
+              <Col span={24}>
+                <StyledFormItem
+                  label={t('Column Data Types')}
+                  name="column_data_types"
+                >
+                  <Input aria-label={t('Column data types')} type="text" />
+                </StyledFormItem>
+                <p className="help-block">
+                  {t(
+                    'A dictionary with column names and their data types if you need to change the defaults. Example: {"user_id":"int"}. Check Python\'s Pandas library for supported data types.',
+                  )}
+                </p>
+              </Col>
+            </Row>
+          </Collapse.Panel>
+          <Collapse.Panel
+            header={
+              <div>
+                <h4>{t('Rows')}</h4>
+                <p className="helper">{t('Adjust row settings.')}</p>
```

Comment:
Suggested text change
`Adjust row settings.` -> 
`Set header rows and the number of rows to read or skip.`